package.json

```diff
@@ -7,17 +7,22 @@
   "devDependencies": {
     "@types/benchmark": "1.0.30",
     "@types/debug": "0.0.29",
-    "@types/node": "7.0.18",
-    "@types/ramda": "0.0.9",
-    "codecov": "2.2.0",
-    "lerna": "2.0.0",
-    "nodemon": "1.11.0",
-    "tap": "10.7.0",
-    "ts-node": "3.0.2",
-    "tslint": "5.1.0",
-    "tslint-eslint-rules": "4.0.0",
-    "tslint-language-service": "0.9.3",
-    "typescript": "2.4.1"
+    "@types/node": "^8.0.11",
+    "@types/ramda": "^0.24.3",
+    "codecov": "^2.2.0",
+    "gulp": "^3.9.1",
+    "gulp-sourcemap": "^1.0.1",
+    "gulp-tslint": "^8.1.1",
+    "gulp-typescript": "^3.2.0",
+    "gulp-util": "^3.0.8",
+    "lerna": "^2.0.0",
+    "lodash": "^4.17.4",
```

Comment:
Let's drop lodash if it's not absolutely needed (see comment on `gulpfile.js`).

As much as I :heart: lodash, I like to keep the dependencies to an absolute minimum. :)